src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int MAX_NUMBER = 100;
```

Comment:
The `MAX_NUMBER` constant should not be static in this context as per the checklist. Static methods and fields are generally discouraged. Consider removing the static modifier.